src/Root.h

```diff
@@ -70,22 +70,31 @@ class cRoot
 	a_OverworldName should be set for non-overworld dimensions if one wishes that world to link back to an overworld via portals
 	*/
 	cWorld * CreateAndInitializeWorld(const AString & a_WorldName, eDimension a_Dimension = dimOverworld, const AString & a_OverworldName = "");
+
+	/** Returns the up time of the server in seconds */
+	long int GetServerUpTime(void)
+	{
+		return static_cast<long int>(std::chrono::duration_cast<std::chrono::seconds>(std::chrono::steady_clock::now() - m_StartTime).count());
```

Comment:
I'd rather you didn't use long as it causes too many issues with differences between windows and linux, choose either long long, or int.